lib/cryptoexchange/exchanges/novaexchange/models/market_pair.rb

```diff
@@ -0,0 +1,14 @@
+module Cryptoexchange::Exchanges
+  module Novaexchange
+    module Models
+      class MarketPair < Cryptoexchange::Models::MarketPair
+        attr_accessor :name
+
+        def initialize(params={})
+          super
+          @name = params[:name]
```

Comment:
@dominicwong617 seems like a bug to me, the pair name is `ESP_BLRY`, and within the hash, the base is `DOGE`